controller/controller.js

```diff
@@ -144,6 +168,19 @@ module.exports = class Controller extends BaseController {
   _getErrors(req, res, callback) {
     super._getErrors(req, res, () => {
       Object.keys(req.form.errors).forEach(key => {
+        var field = req.form.options.fields[key];
+        // get first option for radios
+        if(field.mixin == 'radio-group') {
+          req.form.errors[key].errorLinkId = key + "-" + field.options[0];
+        }
+        // get first field for date input control
+        else if (field && field.controlType === 'date-input') {
+          req.form.errors[key].errorLinkId = key + '-day';
```

Comment:
this is for adding `-day` so that when a user clicks a link they go to a that specific area for an error